llvm/lib/Transforms/Scalar/InferAddressSpaces.cpp

```diff
@@ -305,10 +305,15 @@ static bool isNoopPtrIntCastPair(const Operator *I2P, const DataLayout &DL,
 }
 
 // Returns true if V is an address expression.
-// TODO: Currently, we consider only phi, bitcast, addrspacecast, and
-// getelementptr operators.
+// TODO: Currently, we consider only arguments and phi, bitcast, addrspacecast,
+// and getelementptr operators.
```

Comment:
too many "and". I'd drop the first one.

Reply:
I agree the current wording is a bit awkward but "arguments, phi, bitcast, addrspacecast, and getelementptr operators." isn't really correct since arguments aren't operators. The current wording is intended to mean we support arguments and these 4 types of operators.

Reply:
Perhaps a list would convey it better, then?

We consider only:
  - arguments
  - phi, bitcast, addrspacecast, and getelementptr operators

Reply:
Yep, that is better, updated.